plan.md

```diff
@@ -1,9 +1,282 @@
+could be kept on at this organisation that dwyl's co-founders were heavily involved
+in building, we [started to take on client projects](https://github.com/dwyl/ateam-proposal)(again)
+whilst building our learning community by sharing the knowledge as we went.
+
+This plan lays out **how dwyl aims to achieve its goals**, _clarifying_ the many hats
+dwyl wears and how they _work together_.
+
+## Contents Guide
+For the time being, we are putting everything into this one .md file, to be split
+out once reviewed from its rough draft form. We'll need a way to navigate!
++ [Context](#context)
++ [Branches](#branches)
+  + [Product](#product)
+  + [Services](#services)
+  + [Training](#training)
+  + [Co-working](#co-working)
+
+
+## Context
+
+![tree-branches-image](http://www.m-a.org.uk/features/big/branches_main.jpg)
+
+2017 is the year of **formalising the various types of work we do into their
+own, neat groupings of activities**, the various _branches_ of dwyl.
+
+> Each **branch** will have it's own mission, detailed goals and timelines,
+feeding directly into dwyl's overall goal.
+
+dwylers will work within one branch for at least 50% of their time, but _will
+contribute to more than one_ depending on personal preferences and dwyl's needs.
+For example, a developer working mostly on product could spend some time doing QA
+on a _services_ project and also film a introductory video tutorial for a new technology they've learned.
+
+## Branches
+All branches **must contribute to our fixed overheads** and/or contribute to our
+credibility as experts in a given field.
+
+Each branch will have **its own set of goals**, derived from [dwyl's guiding principles](github.com/dwyl/phase-two#guiding-principles)
+which will be refined over time to serve as it's _decision making framework_.
+
+> There is overlap across all of our branches - they all present multiple opportunities to feed into each other.
+
+### Product
+Building products is how we serve the _most_ people.
+
+Our intention has always been to be a 'products company'.  
+Building tools that make *our* lives easier and more effective as an
+organisation means that we will be start by solving a problem we not only already
+intimately familiar with, but are passionate about solving.
+
+Market research will be carried out and a business case will be put together for
+the creation of each of our products.
+
+#### Goals
++ Do we use/reach for the product _every day_?
++ Further goals to be set:
+  + Date for MVP
+  + Number of _active_ people by the end of the calendar year (based on repeated interaction with the app, [not time spent on it](https://github.com/dwyl/hq/issues/149#issuecomment-269154087))
+
+#### What?
+
+#### First products
+> Our **first products** will be [time](github.com/dwyl/time) (name TBC!) and [tudo](github.com/dwyl/tudo).
+
+Built with a modular architecture, this application will be a cross between a
+to-do list and a time tracking application.
```

Comment:
_aye lassie_, but if we help _everyone_ focus on what product we need to be spending our time on (_pun intended_) so that we can get it to "_users_" ASAP, it would be a good thing.